### PR TITLE
[IMP] marketing: Update rst for target_audiences.rst (v14)

### PR DESCRIPTION
**Pull request by ksc-odoo**
made necessary RST formatting changes based on decisions made during the Writer's Workshop meeting

**This PR should be FWP to 15.0 & 16.0**

Project Task: https://www.odoo.com/web#id=3329143&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 1]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 
-Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
+Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Then, click on the default
 :guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
 drop-down menu full of filter options. From this drop-down, scroll down (or search for)
 :guilabel:`Type`.
 
-Keep the second portion of the filter equation on the default :guilabel:`= (equal sign)`.
+Keep the second portion of the filter equation on the default :guilabel:` 🟰 ` (equal sign)
+icon.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:64]:
```suggestion
Keep the second portion of the filter equation on the default :guilabel:`🟰 (equal sign)` icon.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 2]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
-Then, change the third (and final) portion of the filter equation from :guilabel:`Lead` to
+Next, change the third (and final) portion of the filter equation from :guilabel:`Lead` to
 :guilabel:`Opportunity`. The number of :guilabel:`Records` that fit this specific filter equation
-will change as the equation is customized.
+changes as the equation is customized.
 
-Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the
-equation.
+Add another node to this filter by clicking the :guilabel:` ➕ ` (plus sign icon) to the right of
+the equation.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:71]:
```suggestion
Add another node to this filter by clicking the :guilabel:`➕ (plus sign)` icon to the right of the
equation.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 3]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 
-With "new" opportunities being the target of this filter, the second node will focus on only
+With "new" opportunities being the target of this filter, the second node will focus on *only*
 locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select
 the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down
 (or search for) :guilabel:`Stage` from the field drop-down menu.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:76]: Can break this line later
```suggestion
the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down (or
search for) :guilabel:`Stage` from the field drop-down menu.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 4]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 
-Once again, leave the second portion of the filter equation on :guilabel:`= (equal sign)`.
+Once again, leave the second portion of the filter equation on :guilabel:` 🟰 ` (equal sign)`.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:78]:
```suggestion
Once again, leave the second portion of the filter equation on :guilabel:`🟰 (equal sign)` icon.

```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 5]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 
-Scenario #2: Narrow the target down to the event attendees that purchased a specific ticket
--------------------------------------------------------------------------------------------
+Scenario #2: Narrow down target to event attendees who purchased a specific ticket
+----------------------------------------------------------------------------------
 
-While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click
-:guilabel:`Search More` from the drop-down menu. Then, scroll down (or search for)
-:guilabel:`Event` and select it.
+While in Edit mode on a campaign template form (by clicking the :guilabel:`Edit` button), select
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:91]:
```suggestion
While in *Edit mode* on a campaign template form (by clicking the :guilabel:`Edit` button), select
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 6]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
+scroll down (or search for) :guilabel:`Event`, and select it.
 
 Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
 :guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
 drop-down menu full of filter options. From this drop-down, scroll down (or search for)
 :guilabel:`Event`.
 
-Click the the default :guilabel:`= (equal sign)` in the second portion of the filter equation.
-This reveals a drop-down menu. On this menu, select :guilabel:`contains`.
+Click the the default :guilabel:` 🟰 ` (equal sign)` in the second portion of the filter equation.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:100]:
```suggestion
Click the default :guilabel:`🟰 (equal sign)` icon in the second portion of the filter equation.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 7]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
+This reveals a drop-down menu. From this drop-down menu, select :guilabel:`contains`.
 
 In the third (and final) empty portion of the filter equation, type in the name of the event(s)
-that you would like Odoo to consider for this campaign filter.
+that Odoo should consider for this campaign filter.
 
-Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the
-equation.
+Then, add another node to this filter by clicking the :guilabel:` ➕ ` (plus sign icon) to the right
+of the equation.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:107]:
```suggestion
Then, add another node to this filter by clicking the :guilabel:`➕ (plus sign)` icon to the right of
the equation.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 8]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 The second node will focus on targeting this campaign to attendees who purchase a specific type of
-ticket to the aforementioned event(s) in the first filter equation. To do that, select the default
-:guilabel:`ID` from the first portion of the second filter equation, and scroll down (or search
-for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in that same drop-down menu,
-select :guilabel:`Name`.
+ticket to the aforementioned event(s) mentioned in the first filter equation.
+
+To do that, select the default :guilabel:`ID` from the first portion of the second filter equation,
+and scroll down (or search for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in that
+same drop-down menu, select :guilabel:`Name`.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:114]: Line needs to break sooner
```suggestion
and scroll down (or search for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in
that same drop-down menu, select :guilabel:`Name`.
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 9]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 
-Once again, click the the default :guilabel:`= (equal sign)` in the second portion of the filter
+Once again, click the the default :guilabel:` 🟰 ` (equal sign)` in the second portion of the filter
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:116]:
```suggestion
Once again, click the default :guilabel:`🟰 (equal sign)` icon in the second portion of the filter
```

File: content/applications/marketing/marketing_automation/getting_started/target_audience.rst [hunk 10]
```diff
@@ -43,78 +45,80 @@ To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another br
...
 equation, and select :guilabel:`contains`.
 
-Lastly, in the third (and final) portion of the second filter equation, which will be blank, type
-in the name of the ticket type that should be used for the filter. In this case,
-:guilabel:`Standard` is the name of the event ticket type for this sample filter.
+Lastly, in the third (and final) portion of the second filter equation, which is blank, type in the name
+of the ticket type that should be used for the filter. In this case, :guilabel:`Standard` is the name of
+the event ticket type for this sample filter.
```
Comment (samueljlieber) [inline content/applications/marketing/marketing_automation/getting_started/target_audience.rst:121]:
```suggestion
Lastly, in the third (and final) portion of the second filter equation, which is blank, type in the
name of the ticket type that should be used for the filter. In this case, :guilabel:`Standard` is
the name of the event ticket type for this sample filter.
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 👋
I've made some technical changes, please review them below and let me know if you approve or have any changes. I will push these changes up after this review. 🙂

**Comment by samueljlieber** [comment]
Implemented my [technical changes](https://github.com/odoo/documentation/pull/4321#pullrequestreview-1411840359) in https://github.com/odoo/documentation/commit/d9609f563aa2e41a063122e24544a783ec2beb0c.

**Review by samueljlieber — APPROVED** [review]

**Comment by samueljlieber** [comment]
Hi @StraubCreative this PR is ready for your review!

**Comment by samueljlieber** [comment]
Rebased to latest and fixed 2 early line breaks in https://github.com/odoo/documentation/commit/442ba9eb7159abce044e2dd39dc07799f2d6d038 :)

**Review by StraubCreative — APPROVED** [review]
LGTM. Thanks all 🙂 

@robodoo r+

**Comment by ksc-odoo** [comment]
> LGTM. Thanks all 🙂
> 
> @robodoo r+

Anything I need to do?

**Comment by StraubCreative** [comment]
@robodoo -

**Pull request closed without merging**